### PR TITLE
Update team_members.csv

### PR DESCRIPTION
**Pull request by rrefoy**
Edited the URL that points to blog posts done by moi to be corrected to my current name.

https://18f.gsa.gov/author/rebecca-refoy-sidibe/

# Pull request summary
Here's what I'm trying to do with this PR: Edited the URL that points to blog posts done by moi to be corrected to my current name.

https://18f.gsa.gov/author/rebecca-refoy-sidibe/ should be https://18f.gsa.gov/author/rebecca-refoy/

## Reminder - please do the following before assigning reviewer

- [ ] Update readme
- [ ] For frontend changes, ensure design review
- [ ] For content changes beyond typos, add Ron Bronson as a reviewer

And make sure that automated checks are ok

- fix houndci feedback
- ensure tests pass
- federalist builds
- no new SNYK vulnerabilities are introduced

### CODE REVIEW

**Comment by cantsin** [comment]
The tests/CI did not catch this but with this change, we get `rebecca-refoy-sidibe is undefined` messages in the output, because that author becomes "unknown". So I think we'll have to change the `author` front matter in the relevant blog posts. I'm not too sure if we can do a `redirect_from` for authors. @beechnut do you know?

**Comment by beechnut** [comment]
@cantsin The last column of the CSV is redirect_from, so, you in fact can do that! It should autogenerate. Also, I'm surprised that the undefined author error didn't break the build — it should have. Let's open an issue soon.

@rrefoy I noticed that you only changed the slug — did you also want to change your display name?

**Comment by nateborr** [comment]
In my local environment, I tried adding a redirect from `/author/rebecca-refoy-sidibe/` via the CSV file, but I don't see any evidence that the `redirect_from` values are being written into the front matter for the generated author pages. @beechnut if that's surprising I can capture a follow-up ticket to look into that (it's possible I'm also interpreting that logic incorrectly).

In any event, to avoid the undefined errors I think we do need to update the author slug to `rebecca-refoy` in all of the existing blog posts. I can push up that change myself sometime this week.

File: content/posts/2016-03-22-what-was-your-path-to-18f.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ _The acquisitions team provides innovative services and solutions to support dig
   “Lots of time in the non-profit world with a focus on setting up systems for people to do their jobs better — not realizing until fairly recently there were entire professions of the skills I was using (user research, user experience, information architecture, curriculum development). After meeting 18F-ers I realized there were big teams of people who loved helping other people do their work better. <3”  - {{ "jessie-posilkin" | team_link }}
   </blockquote>
 </div>
-<div class="pquote">{{ "rebecca-refoy-sidibe" | team_photo }}
```
Comment (beechnut) [inline content/posts/2016-03-22-what-was-your-path-to-18f.md:146]: clarification: I'm not seeing the image file renamed in this PR — have you checked to make sure this image is rendering correctly?

File: content/posts/2016-03-22-what-was-your-path-to-18f.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ _The acquisitions team provides innovative services and solutions to support dig
...
+<div class="pquote">{{ "rebecca-refoy" | team_photo }}
   <blockquote>“My path to 18F started with the Peace Corps. I served in Guinea, West Africa and was evacuated in the midst of the Ebola crisis. Returned Peace Corps Volunteers have one year of Non Competitive Eligibility, which makes the hiring process in the federal government much easier. I met my now-supervisor at a career fair that the Peace Corps Headquarters holds twice a year. Five months after being evacuated, I was starting in the Office of Citizen Services and Innovative Technologies. I had never heard of OCSIT or 18F before that. I have a few family members working in various parts of the government who have always regaled me with stories of the maddening bureaucracy. It was encouraging to see an entire office dedicated to hacking the government and helping them to work better. I have been working 50 percent of my time with 18F since November 2015 and have found many more qualities (transparency, professional development, empowerment, focus on diversity, etc) that are inspiring and are refreshing for the public sector.” – Rebecca Refoy-Sidibe</blockquote>
```
Comment (beechnut) [inline content/posts/2016-03-22-what-was-your-path-to-18f.md:147]: clarification: If we're changing the slug to have the correct name, should we also change the attribution at the end of the quote here?

```diff
- the public sector.” – Rebecca Refoy-Sidibe</
+ the public sector.” – Rebecca Refoy</
```

File: content/posts/2016-03-22-what-was-your-path-to-18f.md (hunk 1 above)
Comment (nateborr) [reply content/posts/2016-03-22-what-was-your-path-to-18f.md:146]: There is no photo for the original author slug in the `assets/img/team` directory, so `team_photo` displays an 18F logo. That behavior is visible in production: https://18f.gsa.gov/2016/03/22/what-was-your-path-to-18f/

The change to the slug here doesn't change any behavior but keeps the author slug naming consistent.

File: content/posts/2016-03-22-what-was-your-path-to-18f.md (hunk 2 above)
Comment (nateborr) [reply content/posts/2016-03-22-what-was-your-path-to-18f.md:147]: I can make that change shortly.

**Review by edwintorres — APPROVED** [review]
I tested it locally and it's working for me.

**Pull request merged**